.github/workflows/uat.yml

```diff
@@ -1,4 +1,4 @@
-name: UAT on PR
+name: UAT ON PR
 on:
   pull_request:
   push:
```

Comment:
We don't need push here if you ask me. UAT needs be build only on the pull_request right?

Reply:
Updating accordingly